src/appendices/working-within-test-code.rst

```diff
@@ -0,0 +1,200 @@
+.. _working-within-test-code:
+
+How to Work within Test Code
```

Comment:
This title is confusing to me just because they aren't writing code within any tests, they are working within "Tested Code". Really minor suggestion.

Reply:
yes, i like that better.